packages/ws-server/src/metrics/cpuGauge.ts

```diff
@@ -0,0 +1,45 @@
+/* -
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { Gauge, Registry } from 'prom-client';
+import os from 'os';
+
+/**
+ * Generates a Prometheus Counter metric for tracking WebSocket method calls.
+ * Removes any existing metric with the same name from the provided registry before creating the new metric.
+ * @param {Registry} register - The Prometheus Registry where the metric will be registered.
+ * @param {any} gaugeInfo - Information of the counter.
+ * @returns {Gauge} Returns a new Counter metric instance.
+ */
+export const generateCpuGauge = (register: Registry, gaugeInfo: any): Gauge => {
```

Comment:
I think it would make it more consistent and more organized if you move this logic into the WsMetricRegistry class in the wsMetricRegistry.ts file.

Reply:
Done.